types/address/store_key.go

```diff
@@ -0,0 +1,33 @@
+package address
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// MaxAddrLen is the maximum allowed length (in bytes) for an address.
+const MaxAddrLen = 255
+
+// LengthPrefixStoreKey prefixes the address bytes with its length, this is used
+// for variable-length components in store keys.
+func LengthPrefixStoreKey(bz []byte) ([]byte, error) {
```

Comment:
So, we are going to use this function not only with  a Store Key. Can we reconsider removing this suffix? Or reconsider the previous proposals? Remember that we are already in the `address` context, so no need to bundle that name in function calls (or anything similar: key,....).
Also: https://talks.golang.org/2014/names.slide#7